specs/http-api/README.md

```diff
@@ -476,5 +481,41 @@ True
 
 ---
 
+## List Balances
+This endpoint is *OPTIONAL*. It is only relevant for PFIs which expose a stored balance functionality either for institutional or retail customers.
+
+### Description
+Returns an array of balance resources for each currency held by the caller.  
+
+### Protected
+True
+
+### Endpoint
+`GET /balances`
+
+### Response
+| Status             | Body                                   |
+| ------------------ | -------------------------------------- |
+| `200: OK.     `    | `{ data: { Balance[] } }` |
+| `400: Bad Request` | `{ errors: Error[] }`                  |
+| `404: Not Found`   | N/A                                    |
+| `403: Forbidden`   | N/A                                    |
```

Comment:
why 403 instead of 401?

Reply:
ah wait i just read up more on the difference and i agree now. 401 == "idk u", 403 == "get outta here jerry"

Reply:
yeah 403 is "don't even try - you don't have access". 401 usually means you need to identify yourself and you may get access. Sometimes 404 is also used as you don't want to give away negative existence of a resource (but in this case it is supported or not, so not giving anything away).